ynr/apps/uk_results/management/commands/import_2010_parl_results.py

```diff
@@ -82,4 +120,30 @@ def handle(self, **options):
             ).strip()
             candidate = self.find_candidacy(ballot=ballot, name=candidate_name)
 
-        self.stdout.write(f"{self.cant_find} CANDIDATES COULDNT BE FOUND")
+            if not candidate:
+                continue
+
+            num_ballots = int(row["Vote"].replace(",", ""))
+            candidate_data = {
+                "membership": candidate,
+                "num_ballots": num_ballots,
+                "is_winner": row["Elected"] == "*",
+            }
+            # are we still on same ballot?
```

Comment:
Does this rely on the sheet being ordered properly? Is that ok?

Reply:
You're right, I hadn't considered this. I've refactored now to use a `defaultdict` to remove reliance on the ordering of the sheet